Update JMX Bridge docs to support smoke test addition to tile product

kcboyle opened the pull request:
* Do not merge PR until a goahead is given from Amber Alston
[#132114775]

Commit:
Update JMX Bridge docs to support smoke test addition to tile product
* Do not merge PR until a goahead is given from Amber Alston
[#132114775]

kcboyle commented:
Please do not merge PR until a goahead is given from @AmberAlston

sleslie23 commented:
Hello @kicombs. I blocked this story and added a note. We will not take any further action until we hear from @AmberAlston. Thanks!

AmberAlston committed:
Update deploy-metrics.html.md.erb

Correction to PR that doesn't make sense anymore given that we are actually enforcing the ERT dependency now. Chatted with Kira before making this edit.

sleslie23 commented:
Thanks, @kicombs !!!
